project/Dependencies.scala

```diff
@@ -25,6 +25,7 @@ object Dependencies {
     val playJava           = "com.typesafe.play" %% "play-java"             % Versions.play // Apache V2
     val playGuice          = "com.typesafe.play" %% "play-guice"            % Versions.play // Apache V2
     val playAkkaHttpServer = "com.typesafe.play" %% "play-akka-http-server" % Versions.play // Apache V2
+    val playAkkaHttp2Support = "com.typesafe.play" %% "play-akka-http2-support" % Versions.play // Apache V2
```

Comment:
Play users that want to use gRPC will need to use `play-akka-http2-support` so we add it as a dependency on `play-grpc` since we can assume it'll always be available.